Adyen/Components/Base/PaymentComponentData.swift

```diff
@@ -69,6 +69,9 @@ public struct PaymentComponentData: ShopperInformation {
     
     /// Indicates the device default browser info.
     public let browserInfo: BrowserInfo?
+
+    /// The billing address information.
+    public let billingAddress: AddressInfo?
```

Comment:
maybe add `AddressInfo` to `ShopperInformation` protocol?

Reply:
Good one!